frame/contracts/src/wasm/runtime.rs

```diff
@@ -73,6 +73,9 @@ pub enum ReturnCode {
 	/// The call dispatched by `seal_call_runtime` was executed but returned an error.
 	#[cfg(feature = "unstable-interface")]
 	CallRuntimeReturnedError = 10,
+	/// Recovery failed for other reasons. Most probably wrong recovery id or signature.
+	#[cfg(feature = "unstable-interface")]
+	RecoveryFailed = 11,
```

Comment:
```suggestion
	/// Recovery failed for other reasons. Most probably wrong recovery id or signature.
	#[cfg(feature = "unstable-interface")]
	RecoveryFailed = 11,
```
```suggestion
	/// ECDSA pubkey recovery failed. Most probably wrong recovery id or signature.
	#[cfg(feature = "unstable-interface")]
	RecoveryFailed = 11,
```

The "other" makes no sense here as we do not mention what are those other reasons are.